pilot/pkg/serviceregistry/kube/controller/namespacecontroller.go

```diff
@@ -132,7 +131,21 @@ func NewNamespaceController(data func() map[string]string, options Options, kube
 	})
 	c.configMapController = configmapInformer
 
-	namespaceInformer := informer.NewNamespaceInformer(kubeClient, options.ResyncPeriod, cache.Indexers{})
+	namespacemlw := listwatch.MultiNamespaceListerWatcher(watchedNamespaceList, func(namespace string) cache.ListerWatcher {
```

Comment:
This won't work as namespaces are cluster-scoped. I think we'll need to remove the watching of namespaces entirely

Reply:
Looks like this is used for watching namespaces and creating/updating a configMap with a certificate. We should probably move that into the operator. I filed https://issues.redhat.com/browse/MAISTRA-1724 to take care of it

Reply:
Yeah, I figured this was pretty cheesy.  I think we had something like this in 1.1 though, although maybe mlw worked differently there (that's where I pulled it from).  I was considering adding a listener to the member roll controller, but that required a bit more work to hook it in. (Basically, have the listener add events to the queue for each namespace in the list.)

Reply:
Yeah, that probably makes the most sense - use the NamespaceUpdated events from the MRC and pipe it into this ConfigMap-creating code